### ISSUE TITLE
feat: ✨ consistent naming for "placement" attribute

### ISSUE THREAD

**Issue by MartaPintoTeixeira** [solid-design-system/solid#1583]
## Description
At the moment we are using different names for placement (eg. placement/position/alignment)
In order to keep consistency we want to update the variant names to "placement".

Components that need variant renaming:
- sd-datepicker "position" (left...) 
- sd-link has "alignment" (bottom right...)
- sd-container has "position" (triangle top...)

### Open Questions towards development
- 

## DoR
- [ ] Item has been estimated by the team
- [ ] Item dependencies have been identified and documented

## DoD
- [ ] Brand approval (if neccessary)
- [ ] Figma documentation file incl. hand-off has been created/updated (if applicable)
- [ ] Tokens have been applied/updated (if applicable)
- [ ] Crosslinks are created/updated
- [ ] Implementation works successfully on `figma` branch

**Comment by coraliefeil** [comment]
@van-nguyeb-ht is this sth that already came up during your audit? Is the topic described here still relevant? I remember that we used prop names according to Shoelace. So we de. need to align with dev again on this topic.

**Comment by van-nguyen-ht** [comment]
@coraliefeil yeah I think this should be aligned with dev since datepicker, link & container is not found straight forward in Shoelace so the prop didn't have reference there. 

Should I add SP and process this?
